gradle.properties

```diff
@@ -16,7 +16,7 @@ jtsVersion=1.19.0
 jUnitVersion=5.9.1
 keycloakVersion=19.0.1
 kotlinVersion=1.7.10
-ktfmtVersion=0.41
+ktfmtVersion=0.42
```

Comment:
This is the only manual edit. Everything else is from `./gradlew spotlessApply`.